packages/playground/.eslintignore

```diff
@@ -0,0 +1 @@
+packages/*/src/exports.gen.ts
```

Comment:
Can you check if you have EditorConfig set up? This should have a trailing newline